core/types/wo.go

```diff
@@ -444,7 +444,7 @@ func (wo *WorkObject) TransactionsWithReceipts() []*Transaction {
 			// ignore the coinbase tx
 			continue
```

Comment:
i thought you added receipts to all txs? so the coinbase txs dont have receipts?

Reply:
This function doesn't exist anymore